src/test/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpointIT.java

```diff
@@ -6,11 +6,7 @@
 
 import static com.google.common.collect.Lists.newArrayList;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.hasItem;
-import static org.hamcrest.Matchers.hasProperty;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.*;
 import static org.mockito.Matchers.anyObject;
```

Comment:
Not sure checkstyle would complain about the use of `*`

Reply:
annoying but somehow Codecov does. Actually, I was expecting your return for this :D